pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -387,6 +387,11 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 					Resources: util.NewStringSet("services"),
 				},
 				{
+					// Needed to get namespace<->id mappings for service isolation rules
```

Comment:
has compatibility implications, need to update release notes to note new required permission